README.md

```diff
@@ -11,14 +11,16 @@
 
 ## Introduction
 
-`avail-light` is a data availability light client that can do the following:
+Naive approach for building one AVAIL light client, which will do following
```

Comment:
@kroos47 I think the doc team changed the wordings here. Kindly keep the changes proposed by them.